mixnode/src/commands/describe.rs

```diff
@@ -1,35 +1,23 @@
 use crate::commands::*;
 use crate::config::Config;
 use crate::node::node_description::NodeDescription;
-use clap::{App, Arg, ArgMatches};
-use colored::Colorize;
 use config::NymConfig;
 use std::io;
 use std::io::Write;
 
-pub fn command_args<'a, 'b>() -> App<'a, 'b> {
-    App::new("describe")
-        .about("Describe your mixnode and tell people why they should delegate stake to you")
-        .arg(
-            Arg::with_name(ID_ARG_NAME)
-                .long(ID_ARG_NAME)
-                .help("The id of the mixnode you want to describe")
-                .takes_value(true)
-                .required(true),
-        )
+#[derive(Args)]
+pub(crate) struct Describe {
+    /// The id of the mixnode you want to describe
+    #[clap(long)]
+    id: String,
```

Comment:
This comment really applies to all argument parsing, but I don't feel like I have to copy-paste it everywhere : ) 
would it make sense to "restore" the `about`? In `clap` 3.0 it's applied via `help` macro, so in that case it'd be something like: `[clap(long, help = "Describe your mixnode....")]`

Reply:
Isn't this picked up from the doc comment? The string you're referring to is on the subcommand enum